CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Mattermost Desktop Application Changelog
 
+## Release v3.7.1
+
+Release date: August 30, 2017
+
+This release contains a security update and it is highly recommended that users upgrade to this version.
```

Comment:
I'm not picky on the language, I just wondered if you wanted to explicitly mention Electron or link to the [Electron project's post](https://github.com/electron/electron/releases/tag/v1.6.11) on the security update. If not, I'm fine with the current language.